sympy/series/tests/test_limits.py

```diff
@@ -652,10 +652,18 @@ def test_issue_18306():
     assert limit(sin(sqrt(x))/sqrt(sin(x)), x, 0, '+') == 1
 
 
+def test_issue_18378():
+    assert limit(log(exp(3*x) + x)/log(exp(x) + x**100), x, oo).doit() == 3
```

Comment:
Why `doit()`?

Reply:
Yeah actually not needed. Will change it asap. Anything else?